src/components/screens/PointOfInterest.js

```diff
@@ -210,11 +210,20 @@ export const PointOfInterest = ({ data, hideMap, navigation, route }) => {
                 [category.iconName || MAP.DEFAULT_PIN]: 1,
                 iconName: category.iconName || MAP.DEFAULT_PIN,
                 id,
-                position: { latitude, longitude }
+                position: { latitude, longitude },
+                serviceName: title,
+                title
```

Comment:
Do we really need these two properties? After all, `serviceName` and `title` properties are valid for CalloutText. But we do not use CalloutText in the POI map.

Reply:
This section has been removed as it was unnecessary.